src/test/compile-fail/issue-28433.rs

```diff
@@ -8,16 +8,12 @@
 // option. This file may not be copied, modified, or distributed
 // except according to those terms.
 
-use zoo::bird::{duck, goose};
-
-mod zoo {
-    pub enum bird {
-        pub duck, //~ ERROR: unnecessary `pub` visibility
-        goose
-    }
+enum bird {
+    pub duck, //~ ERROR: expected identifier, found keyword `pub`
```

Comment:
This test will not pass because of https://github.com/rust-lang/rust/issues/28439 I think.